openfisca_us/variables/gov/irs/taxable_income/deductions/standard_deduction/basic_standard_deduction.py

```diff
@@ -0,0 +1,41 @@
+from openfisca_us.model_api import *
+
+
+class basic_standard_deduction(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Basic standard deduction"
+    definition_period = YEAR
+    unit = USD
+    reference = "https://www.law.cornell.edu/uscode/text/26/63#c_2"
+
+    def formula(tax_unit, period, parameters):
+        std = parameters(period).irs.deductions.standard
+        filing_status = tax_unit("filing_status", period)
+        separate_filer_itemizes = tax_unit("separate_filer_itemizes", period)
+        claimed_as_dependent_elsewhere = tax_unit(
```

Comment:
Why is this is a tax-unit-level variable?

Reply:
Hmm, I think because it's a taxcalc variable. Filed https://github.com/PolicyEngine/openfisca-us/issues/773